client/src/components/DataEntry/Ballot.tsx

```diff
@@ -73,14 +140,98 @@ const Ballot: React.FC<IProps> = ({
   )
   const ballot = ballots[ballotIx]
 
-  const [auditing, setAuditing] = useState(true)
   const [interpretations, setInterpretations] = useState<
     IBallotInterpretation[]
   >(contests.map(emptyInterpretation))
+  const { confirm, confirmProps } = useConfirm()
+
+  const interpretiationsRef = useRef<IBallotInterpretation[]>(
+    contests.map(emptyInterpretation)
+  )
+  interpretiationsRef.current = interpretations
+
+  const renderInterpretation = (
+    { interpretation, choiceIds }: IBallotInterpretation,
+    contest: IContest
+  ) => {
+    if (!interpretation) return <div />
+    switch (interpretation) {
+      case Interpretation.VOTE:
+        return contest.choices.map(choice =>
+          choiceIds.includes(choice.id) ? (
+            <h3 key={choice.id}>{choice.name}</h3>
+          ) : null
+        )
+      case Interpretation.BLANK:
+        return <h3>Blank vote</h3>
+      case Interpretation.CONTEST_NOT_ON_BALLOT:
+        return <h3>Not on Ballot</h3>
+      // case Interpretation.CANT_AGREE:
+      // case for Interpretation.CANT_AGREE in case we decide to put it back in again
+      // return (
+      //   <LockedButton disabled large intent="primary">
+      //     Audit board can&apos;t agree
+      //   </LockedButton>
+      // )
```

Comment:
might be worth just removing this since the actual LockedButton part is now out of date. can leave a comment that we don't support CANT_AGREE